gen/geometry/Translation2d.yml

```diff
@@ -36,8 +36,23 @@ classes:
 
 inline_code: |
   cls_Translation2d
+    .def_static("fromFeet", [](units::foot_t x, units::foot_t y){
+      return Translation2d(x, y);
+    })
     .def_property_readonly("x", &Translation2d::X)
     .def_property_readonly("y", &Translation2d::Y)
+    .def_property_readonly("x_feet", [](Translation2d * self) -> units::foot_t {
+      return self->X();
+    })
+    .def_property_readonly("y_feet", [](Translation2d * self) -> units::foot_t {
+      return self->Y();
+    })
```

Comment:
These seem odd. Surely they should be getter methods instead?

Reply:
They're in the same vein as x, so provided for consistency.

Reply:
Ehh. These convert though, the others don't. I really only added those because the method names are rather silly and I didn't want to break anyone who was already using them.